src/pages/dev/general-message-passing/examples.mdx

```diff
@@ -1,9 +1,25 @@
-#  Example cross-chain dApps using GMP
+import { Callout } from "/src/components/callout";
```

Comment:
By convention the title is the first line after frontmatter. We could change this, but we should change this globally in a separate PR.

Reply:
Still unsure on what is effecting my prettier. I will take a deeper look this week at my internal settings for it. Addressed the H1 tag and settings.json in latest commit